packages/core-mobile/app/screens/rpc/components/v2/BridgeAsset.tsx

```diff
@@ -27,17 +29,17 @@ const BridgeAsset = (): JSX.Element => {
   const { goBack } = useNavigation<BridgeAssetScreenProps['navigation']>()
   const { request, asset, amountStr, currentBlockchain } =
     useRoute<BridgeAssetScreenProps['route']>().params
-
   const { onUserApproved: onApprove, onUserRejected: onReject } =
     useDappConnectionV2()
-  const { activeNetwork } = useNetworks()
-
+  const { activeNetwork, networks } = useNetworks()
+  const bridgeAppConfig = useSelector(selectBridgeAppConfig)
   const theme = useContext(ApplicationContext).theme
-
-  const { data: networkFee } = useNetworkFee(activeNetwork)
```

Comment:
we can't just use active network here anymore. for example, when users try to bridge some btc to avalanche via core web and the active network in mobile is avalanche, we don't want to use avalanche's network fee for this transaction. instead we need to infer the correct network to use based on the blockchain param in the request.